src/qlever-petrimaps/server/Server.cpp

```diff
@@ -1108,6 +1112,18 @@ util::http::Answer Server::handleExportReq(const Params& pars, int sock) const {
   return aw;
 }
 
+util::http::Answer Server::handleLoadStatusReq(const Params& pars) const {
```

Comment:
When testing the app, I noticed that the percentage often "jumps back" to something like 60% shortly after it has reached 100%. I am not sure why this is happening, though.

Reply:
Ah, I just noticed that you already gave an explanation for this above :) The `parseId` stage is generally quite short, maybe we could cheat a bit and use the first 95% for the geometry progress, and the last 5% for the IDs.

Reply:
If there is no value of telling the user in which stage of loading they currently are and there won't be any additional stages in the future I think that would be fine. Or maybe it is nice for debugging to let the user know in which stage they were if something goes wrong?